scripts/spack_packages/umpire/package.py

```diff
@@ -189,20 +189,26 @@ def initconfig_hardware_entries(self):
             rocm_root = hip_root + "/.."
             entries.append(cmake_cache_path("HIP_ROOT_DIR",
                                         hip_root))
-            entries.append(cmake_cache_path("HIP_CLANG_PATH",
+            entries.append(cmake_cache_path("ROCM_ROOT_DIR",
+                                        rocm_root))
+            entries.append(cmake_cache_path("HIP_PATH",
                                         rocm_root + '/llvm/bin'))
+            #entries.append(cmake_cache_path("HIP_CLANG_PATH",
+            #                            rocm_root + '/llvm/bin'))
             entries.append(cmake_cache_string("HIP_HIPCC_FLAGS",
                                         '--amdgpu-target=gfx906'))
-            entries.append(cmake_cache_string("HIP_RUNTIME_INCLUDE_DIRS",
-                                        "{0}/include;{0}/../hsa/include".format(hip_root)))
-            hip_link_flags = "-Wl,--disable-new-dtags -L{0}/lib -L{0}/../lib64 -L{0}/../lib -Wl,-rpath,{0}/lib:{0}/../lib:{0}/../lib64 -lamdhip64 -lhsakmt -lhsa-runtime64".format(hip_root)
+            #entries.append(cmake_cache_string("HIP_RUNTIME_INCLUDE_DIRS",
+            #                            "{0}/include;{0}/../hsa/include".format(hip_root)))
+            #hip_link_flags = "-Wl,--disable-new-dtags -L{0}/lib -L{0}/../lib64 -L{0}/../lib -Wl,-rpath,{0}/lib:{0}/../lib:{0}/../lib64 -lamdhip64 -lhsakmt -lhsa-runtime64".format(hip_root)
```

Comment:
Is there a reason to keep these commented lines?  I suggest we remove them or add another comment explaining why they are being left here.